main/src/ca/uwaterloo/flix/api/lsp/provider/completion/AutoImportCompleter.scala

```diff
@@ -54,11 +54,35 @@ object AutoImportCompleter {
   private def javaClassCompletionsByClass(prefix: String, ap: AnchorPosition, env: LocalScope)(implicit root: Root): Iterable[AutoImportCompletion] = {
     val availableClasses = root.availableClasses.byClass.m.filter(_._1.exists(_.isUpper))
     availableClasses.keys.filter(_.startsWith(prefix)).flatMap { className =>
-      availableClasses(className).collect { case path if (!env.m.contains(className)) =>
-        val completePath = path.mkString(".") + "." + className
-        val label = s"$className (import $completePath)"
-          AutoImportCompletion(label, className, completePath, ap, Some(completePath))
+      availableClasses(className).collect { case namespace if (!env.m.contains(className)) =>
+        val qualifiedName = namespace.mkString(".") + "." + className
+        val priority = mkPriotiry(qualifiedName)
+        val label = s"$className (import $qualifiedName)"
+          AutoImportCompletion(label, className, qualifiedName, ap, Some(qualifiedName), priority)
       }
     }
  }
+
+  /**
+    * Returns the priority of the completion item based on the qualified name.
+    *
+    * We will those packages a Low priority:
+    *  - java.lang
+    *  - java.io
+    *  - java.nio
+    *  - java.util
+    *
+    * And we will give those packages a Lowest priority:
+    * - com.sun
+    * - sun.
+    *
+    * Other packages will have a Lower priority.
+    */
+  private def mkPriotiry(qualifiedName: String): Priority = {
```

Comment:
-> priority (spelling)

Reply:
I think in this case the string is actually `className` -- its a Java class name

Reply:
> I think in this case the string is actually `className` -- its a Java class name

Okay we can change to that, but in the code above we use className to refer to `Math` and qualifiedName to refer to `java.lang.Math`. Maybe we should be consistent? Or other names?

Reply:
I see.